modules/apps/portal-search/portal-search-spi/src/main/java/com/liferay/portal/search/spi/reindexer/IndexReindexerRegistry.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.spi.reindexer;
+
+import java.util.Collection;
+import java.util.Set;
+
+/**
+ * @author Gustavo Lima
+ */
+public interface IndexReindexerRegistry {
```

Comment:
i think i would actually put this in the -api module and annotate with ProviderType
the -spi module is more about interfaces that could/would be implemented by someone wanting to extend liferay

Reply:
But indexReindexer is inside -spi so I can't access it inside -api

Reply:
Can I move it to -api?